src/reducers/filterMarkers.js

```diff
@@ -129,7 +130,7 @@ export default (state = initialState, act) => {
         : {
           ...state,
           selectedPlace: state.allResources[act.selectedPlace],
-          showingInfoWindow: true
+          showingInfoWindow: true // debuggin! why is this here, and not in the other branch? 
```

Comment:
Just checking, does this comment still need to be addressed or can it be removed?

Reply:
Ack (no pun intended!) Thanks for catching this, it was a self-comment while I was working, which has now been removed. 

To clarify, by "branch" I meant the `if` branch in `filterReducers`, which did not set the selected place, while the `else` branch did. After a bit more digging, I could not find any instance where we are actually using the resource ID to set the place, hence the if-else seemed unnecessary, removing it altogether removed the need to worry about setting the selected place in both branches. 

TL;DR - I have removed the if-else and that comment in the reducer, and also transitioned to the `createAction` syntax as part of the recent guidelines. Also, some of the commit changes are being caused by prettier formatting, sorry for any confusion!